diff --git a/node_modules/antd/es/descriptions/Item.d.ts b/node_modules/antd/es/descriptions/Item.d.ts
index d0c0f58..ef3461a 100644
--- a/node_modules/antd/es/descriptions/Item.d.ts
+++ b/node_modules/antd/es/descriptions/Item.d.ts
@@ -8,6 +8,7 @@ export interface DescriptionsItemProps {
     contentStyle?: React.CSSProperties;
     children: React.ReactNode;
     span?: number;
+    colon?: boolean;
 }
 declare const DescriptionsItem: React.FC<DescriptionsItemProps>;
 export default DescriptionsItem;
diff --git a/node_modules/antd/es/descriptions/Row.js b/node_modules/antd/es/descriptions/Row.js
index 4a1b6ab..a01d97d 100644
--- a/node_modules/antd/es/descriptions/Row.js
+++ b/node_modules/antd/es/descriptions/Row.js
@@ -26,6 +26,7 @@ function renderCells(items, _ref, _ref2) {
         _ref3$props$span = _ref3$props.span,
         span = _ref3$props$span === void 0 ? 1 : _ref3$props$span,
         key = _ref3.key;
+        colon = _ref3$props.colon === void 0 ? colon : _ref3$props.colon;        
 
     if (typeof component === 'string') {
       return /*#__PURE__*/React.createElement(Cell, {
