diff --git a/node_modules/galio-framework/src/Avatar.js b/node_modules/galio-framework/src/Avatar.js
index d331fbd..c1bc1b5 100644
--- a/node_modules/galio-framework/src/Avatar.js
+++ b/node_modules/galio-framework/src/Avatar.js
@@ -1,5 +1,5 @@
 import React from 'react';
-import { View, Text, StyleSheet, Image, ViewPropTypes } from 'react-native';
+import { View, Text, StyleSheet, Image } from 'react-native';
 import PropTypes from 'prop-types';
 import { withGalio } from 'theme';
 
@@ -87,7 +87,7 @@ Avatar.propTypes = {
   backgroundColor: PropTypes.string,
   imageProps: PropTypes.object,
   imageStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.array, PropTypes.number]),
-  containerStyle: ViewPropTypes.style,
+  containerStyle: PropTypes.any,
   styles: PropTypes.any,
   theme: PropTypes.any,
 };
diff --git a/node_modules/galio-framework/src/Toast.js b/node_modules/galio-framework/src/Toast.js
index fc3b557..eea05b8 100644
--- a/node_modules/galio-framework/src/Toast.js
+++ b/node_modules/galio-framework/src/Toast.js
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { Dimensions, StyleSheet, Animated, ViewPropTypes } from 'react-native';
+import { Dimensions, StyleSheet, Animated } from 'react-native';
 import PropTypes from 'prop-types';
 // galio components
 import Text from './atomic/ions/Text';
@@ -20,8 +20,8 @@ class Toast extends Component {
       PropTypes.string,
     ]),
     round: PropTypes.bool,
-    style: ViewPropTypes.style,
-    textStyle: ViewPropTypes.style,
+    style: PropTypes.any,
+    textStyle: PropTypes.any,
     styles: PropTypes.any,
     theme: PropTypes.any,
   };
