diff --git a/node_modules/rc-field-form/es/utils/valueUtil.js b/node_modules/rc-field-form/es/utils/valueUtil.js
index 3a43429..0313603 100644
--- a/node_modules/rc-field-form/es/utils/valueUtil.js
+++ b/node_modules/rc-field-form/es/utils/valueUtil.js
@@ -115,6 +115,12 @@ export function defaultGetValueFromEvent(valuePropName) {
     return event.target[valuePropName];
   }
 
+  if (event && Array.isArray(event)) {
+    return event.map(function (e) {
+      return e.nativeEvent ? {} : e;
+    });
+  }
+
   return event;
 }
 /**
diff --git a/node_modules/rc-field-form/lib/utils/valueUtil.js b/node_modules/rc-field-form/lib/utils/valueUtil.js
index c40d37e..462d46f 100644
--- a/node_modules/rc-field-form/lib/utils/valueUtil.js
+++ b/node_modules/rc-field-form/lib/utils/valueUtil.js
@@ -145,6 +145,12 @@ function defaultGetValueFromEvent(valuePropName) {
     return event.target[valuePropName];
   }
 
+  if (event && Array.isArray(event)) {
+    return event.map(function (e) {
+      return e.nativeEvent ? {} : e;
+    });
+  }
+
   return event;
 }
 /**
